src/styl/partials/navbar.styl

```diff
@@ -163,7 +163,7 @@
   
   .site-navbar
     justify-content space-between
-    height 56px // Change
+    height 56px !important // Change
```

Comment:
why use !important?

Reply:
because safari is not listening to the element size in the mediaquery

Reply:
ohhhh ok ok